src/js/player.js

```diff
@@ -2473,6 +2477,32 @@ class Player extends Component {
     return this.techGet_('readyState');
   }
 
+  /**
+   * Get a video track list
+   * @link https://html.spec.whatwg.org/multipage/embedded-content.html#videotracklist
+   *
+   * @return {VideoTrackList} thes current video track list
+   * @method videoTracks
+   */
+  videoTracks() {
+    // cannot use techGet_ directly because it checks to see whether the tech is ready.
+    // Flash is unlikely to be ready in time but videoTracks should still work.
+    return this.tech_ && this.tech_['videoTracks']();
```

Comment:
This will return `undefined` when the tech is not ready. The spec says it should return an empty VideoTrackList in that case which seems nice if it's easy to pull off.